lightning/src/ln/interactivetxs.rs

```diff
@@ -1662,14 +1662,55 @@ impl InteractiveTxConstructor {
 	}
 }
 
+/// Determine whether a change output should be added or not, and if so, of what size,
+/// considering our given inputs, outputs, and intended contribution.
+/// Computes and takes into account fees.
+#[allow(dead_code)] // TODO(dual_funding): Remove once begin_interactive_funding_tx_construction() is used
```

Comment:
nit: function name needs to be changed in this comment. haven't looked elsewhere.

Reply:
Well, it's used by `begin_interactive_funding_tx_construction`, that's why I put that here. Is that confusing? Hopefully this `allow` will not stay here for long :D

Reply:
Oh lol definitely makes sense. Thought it was unintentional due to copy paste.